.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: Build Book
+
+on: [push, pull_request]
```

Maybe we should run only on master ?

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```

What do you think about ?

---

done